GUI/XLSearchTaskWindow.xaml.cs

```diff
@@ -153,7 +159,7 @@ private void UpdateFieldsFromTask(XLSearchTask task)
             maxThreadsTextBox.Text = task.CommonParameters.MaxThreadsToUsePerFile.ToString(CultureInfo.InvariantCulture);
             CustomFragmentationWindow = new CustomFragmentationWindow(task.CommonParameters.CustomIons);
             ckbPepXML.IsChecked = task.XlSearchParameters.WritePepXml;
-            ckbPercolator.IsChecked = task.XlSearchParameters.WriteOutputForPercolator;
+            //ckbPercolator.IsChecked = task.XlSearchParameters.WriteOutputForPercolator;
```

Comment:
Why is this commented out?

Reply:
lei doesn't want people using it

Reply:
Please delete the commented code. Odd to have this one commented but the others deleted.

Reply:
Bigger discussion. I'd like to keep it b/c some people may want to use percolator. Lei thinks since we have PEP, just delete it.